gpage.h

```diff
@@ -137,11 +137,14 @@ namespace gcpp {
 			: 0))
 		, min_alloc(min_alloc_)
 		, storage(std::make_unique<byte[]>(total_size))
-		, inuse(total_size, false)
-		, starts(total_size, false)
+		, inuse(locations(), false)
+		, starts(locations(), false)
```

Comment:
Good catch, thanks.

In general (though it wouldn't help in this specific case), it would be nice to have a separate integral type for locations to catch byte/location mismatch errors. I started doing that, but backed away from it as a-distraction-for-now when it took more than one line. O for strong typedefs...